### DISCUSSION TITLE
If there are many nodes and edges, thejsoncrack may cause the browser to break. How to use the reaflow to display the flow chart when there are many nodes and edges?

### DISCUSSION THREAD

**Discussion by niuzhi** [AykutSarac/jsoncrack.com#406] [Q&A] [answered]
If there are many nodes and edges, thejsoncrack may cause the browser to break. How to use the reaflow to display the flow chart when there are many nodes and edges?

**Comment by AykutSarac** [comment, marked as answer]
You can't unless reaflow is optimized to do so. That's a disadvantage of being dependent on a library, thus I've rebuilt the premium editor using Canvas API. Full freedom.
